include/c#/10/APILoader.cs

```diff
@@ -322,5 +337,5 @@ async void SetArmorData(int equipSlot, OfficialAPI.EquipmentItem item)
 	}
 
 	internal static async ValueTask<StatId> ResolveStatId(OfficialAPI.EquipmentItem item)
-		=> item.Stats != null ? (StatId)item.Stats.Id : await APICache.ResolveStatId(item.Id);
+		=> item.Stats is not null ? (StatId)item.Stats.Id : await APICache.ResolveStatId(item.Id);
```

Comment:
Please don't make arbitrary code style changes. The whole codebase uses `!= null` since that is the way i prefer.